.github/workflows/lint-test.yml

```diff
@@ -1,47 +1,74 @@
-name: Lint, and Test
+name: Lint and Test
```

Comment:
This file got renamed recently to `build-lint-test.yml`.

Also, the `name` should be "Build, Lint, and Test".

Reply:
Ok! Yeah I modified because there isn't actually a build step for the repo yet, but I see why we'd want to keep it standardized